src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .trim()
+    .split(';')
+    .map((style) => style.trim())
+    .reduce((stylesObject, style) => {
+      const [property, value] = style.split(':');
+
+      if (property && value) {
+        return { ...stylesObject, [property.trim()]: value.trim() };
```

Comment:
Add a property to the object, if there is both a key and a value.
```suggestion
        stylesObject[property] = value;
```